b2sdk/v1/api.py

```diff
@@ -169,3 +171,34 @@ def download_file_by_id(
                 raise
         else:
             return downloaded_file
+
+    def list_keys(self, start_application_key_id=None) -> dict:
+        """
+        List application keys.
+
+        :param start_application_key_id: an :term:`application key ID` to start from or ``None`` to start from the beginning
```

Comment:
it should be noted that only a max of `self.DEFAULT_LIST_KEY_COUNT` is going to be returned (unlike v2 which returns all keys, unless you stop iterating, I guess?)